tasks/list-remaining-migrated-removals.ts

```diff
@@ -0,0 +1,207 @@
+import { readFileSync, writeFileSync } from 'fs';
+
+import { task, types } from 'hardhat/config';
+import chalk from 'chalk';
+import { ethers } from 'ethers';
+
+import type { FireblocksSigner } from '../plugins/fireblocks/fireblocks-signer';
+
+interface ListMigratedRemovalsTaskOptions {
+  file: string;
+  outputFile?: string;
+  dryRun?: boolean;
+}
+
+type ParsedListMigratedRemovalsTaskOptions = RequiredKeys<
+  ListMigratedRemovalsTaskOptions,
+  'file'
+>;
+
+export const GET_LIST_MIGRATED_REMOVALS_TASK = () =>
+  ({
+    name: 'list-remaining-migrated-removals',
+    description:
+      'Utility to list migrated removals for sale that were not included in migrated certificates',
+    run: async (
+      options: ListMigratedRemovalsTaskOptions,
+      _: CustomHardHatRuntimeEnvironment
+    ): Promise<void> => {
+      const {
+        file,
+        outputFile = 'listed-migrated-removals.json',
+        dryRun,
+      } = options as ParsedListMigratedRemovalsTaskOptions;
+      const jsonData = JSON.parse(readFileSync(file, 'utf8'));
+      // hre.log({ jsonData });
+
+      const [signer] = await hre.getSigners();
+      const signerAddress = await signer.getAddress();
+      const { getRemoval } = await import('@/utils/contracts');
```

Comment:
why isn't this imported at the top-level?

Reply:
haha no idea... I noticed this happen another time in a different file - it might even have been a careless copilot inclusion